[INFO -- 2018-11-04 19:09:29]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-04 19:09:29]
Initialization strings:
[INFO -- 2018-11-04 19:09:29]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-04 19:09:29]
(2) set args /tmp/lreg/randInput /tmp/lreg/output /tmp/lreg/detected.log
[INFO -- 2018-11-04 19:09:29]
initSignal:0
[INFO -- 2018-11-04 19:09:29]
endSignal:10
[INFO -- 2018-11-04 19:09:29]
faultModel:1
[EXCEPTION -- 2018-11-04 19:09:30]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-11-04 19:09:30]
Backtrace BEGIN:
#0  0x0000555555554ed0 in pow@plt ()
#1  0x0000555555556131 in std::pow<double, int> (__x=15, __y=2) at /usr/include/c++/7/cmath:418
#2  0x0000555555555237 in predict (v=15) at lreg.cpp:50
#3  0x00005555555553fc in partialDerivate (pos=0) at lreg.cpp:74
#4  0x0000555555555503 in adjust (alpha=5.0000000000000002e-05) at lreg.cpp:87
#5  0x00005555555556ac in train (iterations=30000, alpha=5.0000000000000002e-05, N=3, stopsWhenStable=true) at lreg.cpp:120
#6  0x0000555555555a1c in execute (input=0x7fffffffe1f1 "/tmp/lreg/randInput") at lreg.cpp:157
#7  0x0000555555555c60 in main (argc=4, argv=0x7fffffffde48) at lreg.cpp:191

175		4° Line : values y of the training set
176		5° Line : set z of new inputs to predict*/
177	
178	
179	int main(int argc, char* argv[]){
180		
181		if (argc != 4) {
182			cout << "Wrong number of arguments" <<endl;
183			cout <<"  Use as : " << endl; 
184			cout <<'\t'<<argv[0]<<" <input file> "<<"<output file> "<<"<detected log file>"<<endl;

Backtrace END
Memory content before bitflip:0x000x000x000x00
Fault Model: Double bit-flip
base address to flip value: 0x7fffffffd524
address max offset: 4
Memory content after  bitflip:0x000x180x000x00
frame name: adjust
symbol name: i
symbol filename: lreg.cpp
symbol line: 86
value: 6144
value address: 0x7fffffffd524
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2018-11-04 19:09:30]
Fault Injection Successful after 1.3213226795196533s
[INFO -- 2018-11-04 19:09:30]
event type: exit
[EXCEPTION -- 2018-11-04 19:09:30]
exit code: no exit code available
